src/main/java/seedu/partyplanet/logic/parser/EListCommandParser.java

```diff
@@ -119,17 +144,21 @@ public EListCommand parse(String args) throws ParseException {
     private Comparator<Event> getSortOrder(ArgumentMultimap argMap) throws ParseException {
         Optional<String> sortType = argMap.getValue(PREFIX_SORT);
         if (sortType.isEmpty()) {
+            stringSort += "Sorted name ";
```

Comment:
Name sounds like a person imo, maybe "titles" to differentiate? like birthday & date, remark & details